frontend/resourceadm/pages/ResourcePage/ResourcePage.tsx

```diff
@@ -72,9 +72,9 @@ export const ResourcePage = (): React.ReactNode => {
     isLoading: resourceLoading,
   } = useSinlgeResourceQuery(selectedContext, repo, resourceId);
   const { data: sectorsData, isLoading: sectorsLoading } = useResourceSectorsQuery(selectedContext);
-  const { data: losData, isLoading: losLoading } = useResourceThematicAreaLosQuery(selectedContext);
-  const { data: eurData, isLoading: eurLoading } =
-    useResourceThematicAreaEurovocQuery(selectedContext);
+  //const { data: losData, isLoading: losLoading } = useResourceThematicAreaLosQuery(selectedContext);
+  //const { data: eurData, isLoading: eurLoading } =
+  // useResourceThematicAreaEurovocQuery(selectedContext);
```

Comment:
What do you think about deleting this instead if it is not used? I think is better to re-implement this in the future if needed. :)

Reply:
100%! Thanks